.github/workflows/create-release.yml

```diff
@@ -7,15 +7,21 @@ on:
       publish-release:
         description: "Publish Release"
         required: false
-        default: "true"
+        type: boolean
+        default: true
       pre-release:
         description: "Pre-release"
         required: false
-        default: "true"
+        type: boolean
+        default: true
       draft:
         description: "Draft"
         required: false
-        default: "false"
+        type: boolean
+        default: false
+
+permissions:
+  packages: write
```

Comment:
Yeah the permissions are really tricky. I spend some times reading and trying to understand. Basically, on PR from fork everything will be deactivated, and then it will depends on the organization or repo configuration. 

The tricky part is that if you start using like this
```yaml
permissions:
  packages: write
```

then you need to define every permission you need as this will reset all the other default permissions to none. Which is an issue here as a new release can't be created. 

I learnt some stuff!